releasenotes-builder/README.md

```diff
@@ -75,6 +76,14 @@ release notes. For example, ```753bc06``` or just tag ```checkstyle-6.12.1``` .
 
 **twitterProperties** - (optional) properties for connection to Twitter.
 
+**publishTwit** - (optional) publish to mailing list from ```mailing_list.txt```.
+
+**mlistUsername** - (optional) username key for mailing list.
+
+**mlistPassword** - (optional) password key for mailing list.
+
+**mlistProperties** - (optional) properties for publication to mailing list.
```

Comment:
is it a path to property file ? if yes please clearly state that in description.